src/main/resources/application.yaml

```diff
@@ -179,7 +179,7 @@ kafka:
     source-1440m: unified.metrics.json.1440m
 
   consumer:
-    group: influxdb-consumer-group
+    group: ${KAFKA_CONSUMER_GROUP}
```

Comment:
Oh, is this so you can run the app in two or more clusters at the same time without joining each other's group? Seeing this made me realize we need the same for our Salus apps.

Reply:
yeah exactly. :-)